types/src/rsmt2d.rs

```diff
@@ -265,6 +326,13 @@ impl<'de> Deserialize<'de> for ExtendedDataSquare {
     }
 }
 
+/// Returns true if and only if the provided coordinates belongs to Original Data Square
+/// (i.e. first quadrant of Extended Data Square).
+pub fn is_ods_square(row: usize, column: usize, square_len: usize) -> bool {
```

Comment:
do we want to make this public api? I'm not sure

Reply:
It would be useful if our users need to interact with ODS, but I'm also not sure.

Reply:
let's make it `pub(crate)` for now

Reply:
done